hack/conformance.sh

```diff
@@ -17,13 +17,27 @@ export KUBECONFIG
 
 teardown() {
     rv=$?
+
     ./sonobuoy status --json
     ./sonobuoy logs > "${ARTIFACTS}/sonobuoy.log"
+
     results_tarball=$(./sonobuoy retrieve)
     cp "$results_tarball" "${ARTIFACTS}/"
     tar -ztvf "$results_tarball"
-    tar -xvzf "$results_tarball" plugins/e2e/
-    cp -f "$(find plugins/e2e/* -name "*.xml")" "${ARTIFACTS}/"
+
+    # Get each plugin junit report rename from 'junit.xml' to 'junit.<plugin-name>.<file number>.xml',
+    # and move to artifacts directory
+    for plugin in $(./sonobuoy status --json | jq -r '[.plugins[]] | unique_by(.plugin) | .[].plugin'); do
+      tar -xvzf "$results_tarball" "plugins/$plugin/"
+      plugin_artifacts_dir="${ARTIFACTS}/$plugin"
+      mkdir -p "$plugin_artifacts_dir"
+      idx=1
+      for report in $(find "plugins/$plugin/"* -name "*.xml"); do
+        plugin_report="${report/.xml/.${plugin}.${idx}.xml}"
```

Comment:
ok, a little bit more to do: the junit files need to be palced and named liek this:  `${ARTIFACTS}/junit.${something}.xml` to be recognized. Prow will merge all junit files together which match this pattern.

Reply:
On line 36 the junit file is renamed from `junit.xml` to `junit.<plugin name>.<file number>.xml`
I verivied it could be picked by spyglass according to [this pattern](https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/kustom/base/manifests/test_infra/1736ab42da959a5be7de22586b495120000bc4ef/starter-s3.yaml#L78-L80)

I just noticed there is an issue with moving it to the right directory though, 
I will fix it.